src/models/contracts/connection.ts

```diff
@@ -42,7 +42,7 @@ export namespace ConnectionCompleteNotification {
 /**
  * Information about a SQL Server instance.
  */
-export class ServerInfo {
+export class ServerInfo implements IServerInfo {
```

Comment:
I'd just remove this class and replace it with a `ServerInfo` interface in the typings. There isn't any reason this needs to be a class anyways.

Reply:
Makes sense. Removed it.